hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -7030,6 +7030,7 @@ public String toString() {
     protected Cell joinedContinuationRow = null;
     private boolean filterClosed = false;
 
+    protected final byte[] startRow;
```

Comment:
Does this need javadoc/comment? It is NOT the same as the RegionInfo#startRow, right? If so, say so (my first though was why this data member when we have this info in RegionInfo... but this is the scan startRow...)

Reply:
added comment here, thx